style-editor.scss

```diff
@@ -55,6 +55,21 @@ body {
 	}
 }
 
+/** === Editor Changes === */
+
+// Since 2019 left-aligns wide and fullwide blocks, left align the toolbar too.
+.editor-block-list__block[data-align="wide"] .editor-block-contextual-toolbar .editor-block-toolbar.editor-block-toolbar,
+.editor-block-list__block[data-align="full"] .editor-block-contextual-toolbar .editor-block-toolbar.editor-block-toolbar {
+		max-width: none;
+}
+
+.editor-block-list__block[data-align="full"] .editor-block-contextual-toolbar .editor-block-toolbar.editor-block-toolbar {
```

Comment:
I agree this is prone to breaking — let's go ahead and remove it. This PR is still great without it.